andela/vidar-ah-backend
# #164139673 Configure Authors Haven to use PostgreSQL as DB Engine

#### What does this PR do?
It configures Authors Haven to use PostgreSQL as DB Engine.

#### Description of Task to be completed?
Authors Haven was formerly built with MongoDB, and this PR does the following:
- Configures Postgresql database instead of MongoDB.
- Adds `dotenv` dependency to keep environmental variables.
- Adds a script `db-migrate` to run migrations.
- Adds a `.env.sample` file so team members will know what is required for the app to run.

#### How should this be manually tested?
- Add a `.env` file, check `.env.sample` for required details and supply them in the `.env` file.
- Run `npm run db-migrate` to run migrations.

#### Any background context you want to provide?
- Right now, we do not have models.
#### What are the relevant pivotal tracker stories?
#164139673
#### Screenshots (if appropriate)
#### Questions:
N/A
